tidb-binlog/pump_client/client.go

```diff
@@ -49,7 +50,7 @@ const (
 )
 
 var (
-	// Logger is ...
+	// Logger is ..., obsolete now.
 	Logger = log.New()
```

Comment:
just delete it if no use

Reply:
need update tidb's code, and then I will clean all obsolete fields